features/create_note.feature

```diff
@@ -1,3 +1,4 @@
+@javascript
```

Comment:
Why did this one need to run with a js driver?

Reply:
I am not sure, if I run it with BROWSER=headless_chrome it all works fine for me without tagging. I think there is a lot of extra js in the applicaiton.html.erb now. I think we should just switch to headless_chrome and remove all @js tags

Reply:
it says it does not find the needs table...

Reply:
Ah, I think it's because for the notes, it runs through the triage form

Reply:
This passes for me if you add:
```
page.fill_in 'contact_needs_needs_list_0_start_on', with: '01/01/2020' if need == 'Phone triage'
```
to need_steps.rb "I add needs {string}"

Reply:
Without javascript there isn't a default date option set

Reply:
The rack tests run about 5-10x quicker than headless_chrome, so there's an advantage in avoiding that.
Given that we are using this more as an e2e test suite than an acceptance test suite, there's an advantage in that imo.